src/main/resources/db/migration/V057__Add_createdat_date_index_on_envelopes.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX envelopes_createdat_idx ON envelopes((createdat::DATE));
```

Comment:
probably don't need extra pair of braces

Reply:
there are 2 options: ON envelopes((createdat::DATE)) or ON envelopes(date(createdat))
I prefer the first one

Reply:
ah. ok, makes sense